src/pages/resources/featured-sponsors-darkmode.svg.js

```diff
@@ -7,54 +7,128 @@ const baseUrl = import.meta.env.SITE
 
 // Width of the composed SVG
 const overallWidth = 814
-// Maximum height a darklogo may have
+// Maximum height a logo may have
 const maxHeight = 50
-// Maximum width a darklogo may have
+// Lead sponsor height (doubled from regular sponsors)
+const leadSponsorHeight = maxHeight * 2
+// Maximum width a logo may have
 const maxWidth = 200
-// Horizontal padding between darklogos
+// Lead sponsor maximum width (doubled from regular sponsors)
+const leadSponsorMaxWidth = maxWidth * 2
+// Horizontal padding between logos
 const xPadding = 40
-// Vertical padding between rows of darklogos
+// Vertical padding between rows of logos
 const yPadding = 20
+// Additional padding below lead sponsors
+const leadSponsorBottomPadding = 80
 
-/**
- * Build an SVG response body with rows of evenly-spaced sponsor darklogos wrapped in anchors.
- * @returns string
- */
 const buildResponse = () => {
   let images = []
-  let totalHeight = 0
-  let currentX = 0
-  let currentY = 0
-  let rowCount = 1
-
-  featuredSponsors.map((sponsor, index) => {
-    const dimensions = sizeOf("./public/" + sponsor.darklogo)
+  let currentY = 30
+
+  // First process lead sponsors
+  const leadSponsors = featuredSponsors.filter(sponsor => sponsor.isLeading === true)
+  const regularSponsors = featuredSponsors.filter(sponsor => sponsor.isLeading !== true)
+
+  console.log("Processing lead sponsors")
+
+  // Handle lead sponsors (centered, own row)
+  if (leadSponsors.length > 0) {
+    // Get dimensions for lead sponsors
+    const leadSponsorInfo = leadSponsors.map(sponsor => {
+      // Ensure we have a valid logo path
+      const logoPath = sponsor.darkLogo ? `./public${sponsor.darkLogo}` : null
```

Comment:
```suggestion
      const logoPath = sponsor.darklogo ? `./public${sponsor.darklogo}` : null
```

Reply:
Oh dear. Thanks. Caps should be outlawed.